apps/src/templates/studioHomepages/Courses.jsx

```diff
@@ -6,13 +6,14 @@ import { CourseBlocksAll } from './CourseBlocks';
 import CoursesTeacherEnglish from './CoursesTeacherEnglish';
 import CoursesStudentEnglish from './CoursesStudentEnglish';
 import ProtectedStatefulDiv from '../ProtectedStatefulDiv';
-import Responsive from '../../responsive';
-import _ from 'lodash';
 import Button from '@cdo/apps/templates/Button';
 import i18n from "@cdo/locale";
+import styleConstants from '@cdo/apps/styleConstants';
 
 const styles = {
   content: {
+    width: '100%',
+    maxWidth: styleConstants['content-width'],
```

Comment:
Same approach as PR https://github.com/code-dot-org/code-dot-org/pull/19688.  Commit e6222b1da4c5350185baba251e4cef89cc931258 should've been part of that PR!